README.md

```diff
@@ -64,7 +64,7 @@ The latest version of the AEM CIF Project Archetype, requires the below minimum
 
 | Archetype | AEM 6.4 | AEM 6.5 | Magento                   | CIF Connector | Java |
 | --------- | ------- | ------- | ------------------------- | ------------- | ---- |
-| 0.10.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 0.10.0        | 1.8  |
+| 0.11.0    | 6.4.4.0 | 6.5.0   | 2.3.4                     | 1.0.0       | 1.8  |
```

Comment:
I approved too quickly: for the connector, I added the Magento versions: `2.3.4 & 2.3.5` to indicate it also works with Magento 2.3.5

Reply:
Done